### ISSUE TITLE
fsharp parser definition refers to incorrect files

### ISSUE THREAD

**Issue by mattgallagher92** [nvim-treesitter/nvim-treesitter#7227]
The fsharp parser definition in ./lua/nvim-treesitter/parsers.lua refers to the files src/parser.c and src/scanner.c in https://github.com/ionide/tree-sitter-fsharp. Those files don't exist, but there are similar files in fsharp/src in that repo.

I assume that that ./lua/nvim-treesitter/parsers.lua needs to be updated to reflect that?

CC @Nsidorenco

**Comment by clason** [comment]
Yes, that's exactly what this line is for: https://github.com/nvim-treesitter/nvim-treesitter/blob/9d2acd49976e2a9da72949008df03436f781fd23/lua/nvim-treesitter/parsers.lua#L623

This is tested by our CI, so any breaking change would be instantly detected. So there's nothing to do here.

**Comment by mattgallagher92** [comment]
Thanks for the prompt response 🙂 and sorry for the incorrect error report! I was seeing some issues with F# tree-sitter integration yesterday, I guess I misattributed the cause.

When I can afford to risk breaking things again (I've rolled back to an earlier version of nvim-treesitter without F# support), I'll see if I can reproduce the issues that I was having and share the actual error messages.
